lib/modules/platform/gerrit/client.ts

```diff
@@ -1,3 +1,4 @@
+import { gte as semverGte } from 'semver';
```

Comment:
use deep import

Reply:
Like `import semverGte from 'semver/functions/gte';`?

No other module in Renovate does that, so I just changed it to `import semver from 'semver';` to be like everywhere else.